tomopyui/widgets/als_imports.py

```diff
@@ -0,0 +1,459 @@
+from abc import ABC, abstractmethod
+import numpy as np
+import pathlib
+import tifffile as tf
+import tomopy.prep.normalize as tomopy_normalize
+import os
+import json
+import dxchange
+import re
+from tomopy.sim.project import angles as angle_maker
+import olefile
+import pandas as pd
+from tomopyui.backend.io import IOBase, ProjectionsBase, RawProjectionsBase
+
+from tomopyui.widgets import helpers
+from ipyfilechooser import FileChooser
+import logging
+from ipywidgets import *
+from tomopyui._sharedvars import *
+import time
+from tomopyui.widgets.plot import BqImPlotter_Import
+from tomopyui.widgets.imports import ImportBase, UploaderBase
+
+
+class Import_ALS832(ImportBase):
+    """"""
+
+    def __init__(self):
+        super().__init__()
+        self.angles_from_filenames = True
+        self.raw_projections = RawProjectionsHDF5_ALS832()
+        self.raw_uploader = RawUploader_ALS832(self)
+        self.use_raw_button = Button(
```

Comment:
I just updated this in https://github.com/samwelborn/tomopyui/commit/3066a58c5f52658be8093d8235fc4899fcf71d82 so you won't have to copy all this code.

Reply:
removed `enable_raw`, `disable_raw` from `Import_ALS832`

Reply:
Also `self.use_raw_button` and `self. use_prenorm_button ` instantiations